jdaviz/configs/specviz/plugins/parsers.py

```diff
@@ -76,17 +80,65 @@ def specviz_spectrum1d_parser(app, data, data_label=None, format=None, show_in_v
             raise ValueError(f"Length of data labels list ({len(data_label)}) is different"
                              f" than length of list of data ({len(data)})")
 
-    with app.data_collection.delay_link_manager_update():
```

Comment:
With this removed, the rest of the block now falls under `if isinstance(data, SpectrumList):`. Is this intentional?